tox.ini

```diff
@@ -77,3 +79,15 @@ commands = pylint --rcfile=tox.ini apptemplates
 deps =
     django110: Django>=1.10,<1.11
     pylint
+
+[testenv:pylint-py27-django111]
+commands = pylint --rcfile=tox.ini apptemplates
+deps =
+    django111: Django>=1.11,<2.0
+    pylint
+
+[testenv:pylint-py35-django111]
+commands = pylint --rcfile=tox.ini apptemplates
+deps =
+    django111: Django>=1.11,<2.0
+    pylint
```

Comment:
As per my comment above, please remove the sections `[testenv:pylint-py27-django110]` and `[testenv:pylint-py35-django110]`, and update the ones you introduced to Python 3.6.

I hope Pylint is happy with Python 3.6, by the way.  :hushed: